scripts/tools/generate-integrations-json.py

```diff
@@ -73,12 +73,14 @@
     f.write(
         json.dumps(
             result_integrations,
-            indent=4,
+            indent=2,
```

Comment:
To sync style with airflow-site.
https://github.com/apache/airflow-site/blob/a9c9aa9f6c3b64fd5c0c12f02835578457e48649/.pre-commit-config.yaml#L151